src/components/comment/comment.twig

```diff
@@ -113,18 +113,27 @@
       <h{{_section_heading_depth}} class="u-hidden-visually">
         Replies to {{comment.author.name}}
       </h{{_section_heading_depth}}>
+
+      {#
+        This is defined here, so we can pass it into the block of a nested embed.
+        @see https://benfurfie.co.uk/articles/how-to-nest-a-block-in-another-block-in-an-embed-in-twig
+      #}
+      {% set repliesContent %}
+        {% block replies %}
+          {% for child in comment.children %}
+            {% include '@cloudfour/components/comment/comment.twig' with {
+              comment: child,
+              heading_depth: _child_heading_depth,
+            } only %}
+          {% endfor %}
+        {% endblock %}
+      {% endset %}
```

Comment:
I'm open to other ideas if there's a less yucky way to handle this. This needs to be defined after `_child_heading_depth` is set

Reply:
I think this is fine 👍